config/general.go

```diff
@@ -0,0 +1,51 @@
+// Copyright © 2022 Meroxa, Inc.
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//     http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+package config
+
+import (
+	"github.com/conduitio/conduit-connector-oracle/config/validator"
+	"github.com/conduitio/conduit-connector-oracle/models"
+)
+
+// A General represents a general configuration needed to connect to Oracle database.
+type General struct {
```

Comment:
What do you think about using  connection string to connect to oracle?
For example
"<your username>/<your password>@service_name"

Reply:
Did you mean to use a connection string to connect to a database like `localhost:1521/ORCLCDB`?

Reply:
You're right.
The connection string will be like `username/password@address:port/dbname`.